utils/polus-imagej-util/{{cookiecutter.project_slug}}/README.md

```diff
@@ -0,0 +1,28 @@
+# {{ cookiecutter.project_name }}
+
+This WIPP plugin was automatically generated by a utility that searches for
+ImageJ plugins and generates code to run them. For more information on what this
+plugin does, contact one of the authors: Nick Schaub (nick.schaub@nih.gov) or 
+Anjali Taneja (anjali.taneja@axleinfo.com).
```

Comment:
You might want to keep Anjali in the list of authors, but remove her email address. Also add your name and email.